assets/js/components/NotFoundComponent.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of the SocialDocs project (c) 2015 - 2020
+ * 
+ * (c) Ben Bieler (Project Lead) <benjaminbieler2014@gmail.com>
+ *
+ * The Licence file can be found in the root of the directory
+ */
+
+"use strict";
+
+import React from 'react';
+
+/**
+ * Triggered when 404
+ *
+ * @author Ben Bieler <benjaminbieler2014@gmail.com>
+ */
+export default class NotFoundComponent extends React.Component {
+    render() {
+        return (
+            <h1>404 Not Found</h1>
```

Comment:
``return <h1>404 Not Found</h1>`` should do since react doesn't require braces when writing one-line JSX